backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/lowlevel/index/settings/impl/IndexSettings.java

```diff
@@ -49,6 +52,15 @@ public void setAnalysis(Analysis analysis) {
 		return extraAttributes;
 	}
 
+	public Integer getMaxResultWindow() {
+		JsonElement maxResultWindowElement = extraAttributes.get( MAX_RESULT_WINDOW_ATTRIBUTE );
```

Comment:
That's weird. Any reason to remove the more explicit `private Integer maxResultWindow`? The point of `extraAttributes` was to hold, well... extra attributes. The ones we don't care about.

Reply:
Also, it seems to me this complicates `IndexSettingsValidator` unnecessarily.

Reply:
Ok I'm putting back the explicit field....